docs/datatypes/core/zioapp.md

```diff
@@ -79,21 +79,10 @@ object CustomizedRuntimeZIOApp extends ZIOAppDefault {
 
 A detailed explanation of the ZIO runtime system can be found on the [runtime](runtime.md) page.
 
-## Installing Low-level Functionalities
```

Comment:
@khajavi Be sure to incorporate these updates into the docs!

Reply:
Will do!